content/blog/multi-lang-components/index.md

```diff
@@ -18,21 +18,23 @@ Pulumi's infrastructure as code tooling combines the programming languages and t
 infrastructure. Until now, some Pulumi components for cloud infrastructure, like our popular [EKS package], were
 only available in a subset of the languages supported by Pulumi.
 
-Now, with multi-language components, anyone can author a Pulumi component in the language they prefer and make it available
-across all Pulumi languages. To celebrate, we've taken that [EKS package]–previously only available for TypeScript–
-and made it available in all four Pulumi languages: TypeScript, Python, .NET, and Go. Regardless of the language you choose, you can
-[manage EKS clusters] with Pulumi, starting with the [v0.22.0 release of the EKS package]. Check out our Modern Infrastructure
-Wednesday video to see it in action:
+Now, you can use the [EKS package]–previously only available for TypeScript–in all four Pulumi languages: TypeScript, Python, .NET,
+and Go. Regardless of the language you choose, you can [manage EKS clusters] with Pulumi, starting with the [v0.22.0 release of the
+EKS package]. Check out our Modern Infrastructure Wednesday video to see it in action:
 
 {{< youtube "gxLyAr0lUg0?rel=0" >}}
 
 <!--more-->
 
-EKS in all languages is just the start of the multi-language components story. Multi-language components enable all organizations to choose
-the right programming language, or languages, for each team and each product. Your infrastructure or platform teams can
+EKS in all languages is made possible by our ongoing work to enable "multi-language components". With multi-language components,
+a developer can author a Pulumi component in their preferred language and then publish it for all Pulumi languages—without needing
+detailed knowledge of the other Pulumi languages.
+
+EKS in all languages is just the start of this story. Multi-language components will enable all organizations to choose
```

Comment:
the overarching benefit we should weave into here is that this will facilitate easier sharing of components with the entire Pulumi ecosystem of developers who are using different languages

Reply:
Adjusted